packages/server/lib/controllers/auth.controller.ts

```diff
@@ -260,6 +292,79 @@ class AuthController {
             next(error);
         }
     }
+
+    getHostedLogin(req: Request, res: Response, next: NextFunction) {
```

Comment:
not sure what `Hosted` referred to here?

Reply:
Hosted = WorkOs

Reply:
I find our use of Hosted rather confusing, everything is hosted by default.
Is it enterprise, cloud or community? if `hosted = enterprise + cloud` maybe we can come up with a naming more representative like "managed". 
Plus we use `hosted` for self-hosted in the Docker images right now, which is the opposite